june news (#1327)
ropensci/roweb3: content/blog/2026-06-29-news-june-2026/index.md

---
slug: "news-june-2026"
title: rOpenSci News Digest, June 2026
author:
  - The rOpenSci Team
date: '2026-06-29'
tags:
  - newsletter
description: 15 years of rOpenSci; champions program; maintainer month; new packages and package news
params:
  last_newsletter: '2026-05-28'
  doi: "10.59350/312sy-1dt83"
rmd_hash: e618ac2f19667303

---

<!-- Before sending DELETE THE INDEX_CACHE and re-knit! -->

Dear rOpenSci friends, it's time for our monthly news roundup! <!-- blabla --> You can read this post [on our blog](/blog/2026/06/29/news-june-2026). Now let's dive into the activity at and around rOpenSci!

## rOpenSci HQ

### Champions Program update

TODO Yani

> The 2026--2027 cohort is continuing their training activities... and has been formally introduced on our blog! Read all about the [11 new champions](/blog/2026/06/09/champions-2026/).

Reviewer: ```suggestion
The 2026--2027 cohort is continuing their training activities, meeting with their mentors, and starting to work on their packages, and they have been formally introduced on our blog! Read all about the [11 new champions](/blog/2026/06/09/champions-2026/).
```

- Social media cards that actually look good
- PACKAGES.rds support (or: implementing R internals in JavaScript)
- Fancy sort/filter bars in the WebUI
- For the impatient: trigger a sync manually
- Making check results easier to find and share

In other news, R-universe user Tom Palmer also wrote about five things: ["Five tips for managing your R-universe 🚀"](https://remlapmot.github.io/post/2026/runiverse-tips/). You won't believe the fifth one. :wink:

### Yanina Joins the 2026 Sovereign Tech Fellowship

We're excited to share that our Community Manager, Yanina Bellini Saibene, [has been selected as a 2026 Sovereign Tech Fellow](https://www.sovereign.tech/news/meet-the-2026-sovereign-tech-fellows). During the fellowship, she will focus on making open source more accessible through improved contribution guidance, newcomer-focused mini-hackathons, multilingual training resources, and more sustainable localization practices across communities in the R ecosystem. These efforts will build on and extend rOpenSci's work in community building, mentorship, and open science.

### *Quinceañera*: celebrating 15 years together

> In June, we held two community events to mark rOpenSci's 15th anniversary. Across all three sessions, people shared memories of their first contribution, discussed ideas for the next 15 years, and reminded us of how genuinely welcoming rOpenSci *and* it's community are. There's more to come :-) Keep an eye out for what we have planned for the rest of the year.

Reviewer: I still think we need to mention the co-working.  What about this?

> In June, we held two community events to mark rOpenSci's 15th anniversary. Across all three sessions, people shared memories of their first contribution, discussed ideas for the next 15 years, and reminded us of how genuinely welcoming rOpenSci *and* it's community are. There's more to come :-) Keep an eye out for what we have planned for the rest of the year.

Reviewer: ```suggestion
In June, we held two community events and a co-working sesion to mark rOpenSci's 15th anniversary. Across all three sessions, people shared memories of their first contribution, discussed ideas for the next 15 years, and reminded us of how genuinely welcoming rOpenSci *and* it's community are. There's more to come :-) Keep an eye out for what we have planned for the rest of the year.
```

  - [pkgmatch](https://github.com/ropensci/software-review/issues/671), Find R Packages Matching Either Descriptions or Other R Packages. Submitted by [mark padgham](https://mpadge.github.io).

- Two at ['5/awaiting-reviewer(s)-response'](https://github.com/ropensci/software-review/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%225/awaiting-reviewer(s)-response%22):

  - [nycOpenData](https://github.com/ropensci/software-review/issues/750), Convenient Access to NYC Open Data API Endpoints. Submitted by [Christian Martinez](https://github.com/martinezc1).

  - [priorsense](https://github.com/ropensci/software-review/issues/704), Prior Diagnostics and Sensitivity Analysis. Submitted by [Noa Kallioinen](https://github.com/n-kall). (Stats).

- Four at ['4/review(s)-in-awaiting-changes'](https://github.com/ropensci/software-review/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%224/review(s)-in-awaiting-changes%22):

  - [RAMEN](https://github.com/ropensci/software-review/issues/743), RAMEN: Regional Association of Methylome variability with the Exposome and geNome. Submitted by [Erick Navarro-Delgado](https://erick-navarrodelgado.netlify.app).

>   - [logolink](https://github.com/ropensci/software-review/issues/741), An Interface for Running NetLogo Simulations. Submitted by [Daniel Vartanian](http://danielvartan.com).

Reviewer 2: I'm not sure how much we care, but here and in galamm and EpiStrainDynamics we could use https
```suggestion
  - [logolink](https://github.com/ropensci/software-review/issues/741), An Interface for Running NetLogo Simulations. Submitted by [Daniel Vartanian](https://danielvartan.com).
```

  - [rcrisp](https://github.com/ropensci/software-review/issues/718), Automate the Delineation of Urban River Spaces. Submitted by [Claudiu Forgaci](https://github.com/cforgaci). (Stats).

  - [galamm](https://github.com/ropensci/software-review/issues/615), Generalized Additive Latent and Mixed Models. Submitted by [Øystein Sørensen](http://osorensen.no). (Stats).

- Five at ['3/reviewer(s)-assigned'](https://github.com/ropensci/software-review/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%223/reviewer(s)-assigned%22):

  - [ciecl](https://github.com/ropensci/software-review/issues/765), International Classification of Diseases ICD-10/ICD-11 for Chile. Submitted by [Rodolfo Tasso](https://github.com/Rodotasso).

  - [EpiStrainDynamics](https://github.com/ropensci/software-review/issues/763), Infer temporal trends of multiple pathogens. Submitted by [Saras Windecker](http://www.smwindecker.com). (Stats).

>   - [lakefetch](https://github.com/ropensci/software-review/issues/762), Calculate Fetch and Wave Exposure for Lake Sampling Points. Submitted by [jeremylfarrell](https://github.com/jeremylfarrell).

Reviewer 2: Appears twice? Here and below...

Author: Thank you! I went and fixed the labelling.
